pythonStacker/createBSMWeights.py

```diff
@@ -20,7 +21,7 @@ def parse_arguments():
                         help="which event class to check")
     parser.add_argument("-p", "--process", dest="process", action="store", default="TTTT_BSM")
     parser.add_argument("--storage", dest="storage", type=str,
-                        default="Intermediate", help="Path at which the histograms are stored")
+                        default="Intermediate_VO", help="Path at which the histograms are stored")
```

Comment:
Slightly worrying for other things down the line, why is this done? Cause everything down the line just uses the generic storage argument afaik, so this just raises questions

Reply:
Because I store each model in a different Intermediate_<> folder; at the beginning each one of them had its own problems and I wanted to keep it separate as much as possible. Still not needed for everybody, it's a bookkeeping solution that worked for me